src/test/pmem2_source/pmem2_source.c

```diff
@@ -232,6 +232,7 @@ test_pmem2_src_mcsafe_write(const struct test_case *tc, int argc,
 
 	size_t bufsize = 4096;
 	void *buf = MALLOC(bufsize);
+	memset(buf, '7', bufsize);
 	ret = pmem2_source_pwrite_mcsafe(src, buf, bufsize, 0);
```

Comment:
I suggest to check if it is really written at the end. (I mean - if pmem2_source_pwrite_mcsafe implementation would be "return 0" this test still pass)